ocs_ci/ocs/platform_nodes.py

```diff
@@ -327,11 +320,51 @@ def restart_nodes(self, nodes, force=True, timeout=300, wait=True):
                     "get events -A --field-selector involvedObject.name="
                     f"{node.name},reason=Rebooted -o yaml"
                 )
+                try:
+                    for node_reboot_events in TimeoutSampler(
+                        timeout=300, sleep=3, func=self.get_reboot_events, nodes=[node]
+                    ):
+                        if (
+                            node_reboot_events[node.name]
+                            != num_events_pre_reboot[node.name]
```

Comment:
shouldn't this be > instead of != ?

Reply:
both works fine me and didn't see any issue with not equal as reboot events will not decrease